### DISCUSSION TITLE
I got partial decoding for Tian BMS used in SacredSun SCIFP48100 LiFePo4

### DISCUSSION THREAD

**Discussion by wire67** [ai-republic/bms-to-inverter#77] [Ideas]
If you're interested I got partial decoding for Tian BMS used in SacredSun SCIFP48100 LiFePo4, I'm using ESPHome as it better suits my needs now but could be useful for someone else in bms-to-inverter as well. 

https://github.com/wire67/esphome_sacredsun_rs485

**Comment by ai-republic** [comment]
Hi @wire67, thanks for the link! I'll integrate the RS485 SacredSun binding maybe on the weekend if I have time. If I have questions I'll be sure to contact you 😄 

I'll move this over to discussion though as it is not an issue 😉

**Comment by ai-republic** [comment]
@wire67 I've implemented the SacredSun RS485 binding according to your experimental implementation. Did you get any futher in decoding the rest of the values?
